modules/switch/react/lib/Switch.tsx

```diff
@@ -63,6 +63,7 @@ const SwitchInput = styled('input')<SwitchProps>(
     marginLeft: spacing.xxxs,
     borderRadius: borderRadius.circle,
     opacity: 0,
+    display: 'block', // make click target width of input
```

Comment:
I don't think this comment is quite accurate. 1) It doesn't change the width and 2) this _is_ the input 😄 

This is makes the input a block element which fixes this by pinning the input to the left (in left-to-right that is)